alphas/alpha_plot.py

```diff
@@ -0,0 +1,16 @@
+import alpha_functions as alphas
```

Comment:
remove this file from this PR. we will use a different approach for this. i will explain it to you offline